components/contact/ContactInputs.tsx

```diff
@@ -0,0 +1,114 @@
+"user client";
+
+import toast from "react-hot-toast";
+import emailjs from "@emailjs/browser";
+import { useRef, MutableRefObject, useState } from "react";
+
+import InputField from "@/components/contact/InputField";
+
+const ContactInputs = () => {
+  const form: MutableRefObject<HTMLFormElement | null> = useRef(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  // NOTE - Cannot make it exportable because of the use of IsLoading
+  const inputField = [
+    {
+      label: "What's your name?",
+      id: "name",
+      name: "name",
+      type: "text",
+      maxLength: 50,
+      required: true,
+    },
+    {
+      label: "What's your email?",
+      id: "email",
+      name: "email",
+      type: "email",
+      maxLength: 500,
+      required: true,
+    },
+    {
+      label: "Write something about your project goals and time frame",
+      id: "goalsTimeFrame",
+      name: "goalsTimeFrame",
+      type: "text",
+      maxLength: 500,
+      required: true,
+      isTextArea: true,
+    },
+    {
+      label: (
+        <>
+          How to reach out to you back?
```

Comment:
"How to reach out to you back" is hard to understand and should be replaced by "Contact me" or a more straightforward sentence like "Reach out" or even "How to reach out".